docs/feeds.md

```diff
@@ -85,22 +82,23 @@ For feeds where a webhook is not available, but do not need high-volume or low l
 
 To set up a polling-based feed, the feed action takes the following steps when called for `CREATE`:
 1.   The feed action sets up a periodic trigger (*T*) with the desired frequency, using the `whisk.system/alarms` feed.
-2.   The feed developer creates a `pollMyService' action which simply polls the remote service and returns any new events.
-3.  The feed action sets up a *rule*
-*T -> pollMyService*.
+2.   The feed developer creates a `pollMyService` action which simply polls the remote service and returns any new events.
+3.  The feed action sets up a *rule* *T -> pollMyService*.
 
-That's it.   This procedure implements a polling-based trigger entirely using OpenWhisk actions, without any need for a separate service.
+This procedure implements a polling-based trigger entirely using OpenWhisk actions, without any need for a separate service.
 
 # Implementing Feeds via Connections
 
 The previous 2 architectural choices are simple and easy to implement. However, if you want a high-performance feed, there is no substitute for persistent connections and long-polling or similar techniques.
 
-Since OpenWhisk actions are stateless,  an action cannot maintain a persistent connection to a third party. Instead, we are forced to stand up a separate service (outside of OpenWhisk) that runs all the time.   We call these *provider services*.  A provider service can maintain connections to third party event sources that support long polling or other connection-based notifications.   
+Since OpenWhisk actions must be short-running,  an action cannot maintain a persistent connection to a third party . Instead, we must
+stand up a separate service (outside of OpenWhisk) that runs all the time.   We call these *provider services*.  A provider service can maintain connections to third party event sources that support long polling or other connection-based notifications.   
 The provider service should provide a REST API that allows the OpenWhisk *feed action* to control the feed.   The provider service acts as a proxy between the event provider and OpenWhisk -- when it receives events from the third party, it sends them on to OpenWhisk by firing a trigger.
 
 The Cloudant *changes* feed is the canonical example -- it stands up a `cloudanttrigger` service which mediates between Cloudant notifications over a persistent connection, and OpenWhisk triggers.
 <!-- TODO: add a reference to the open source implementation -->
 
 The *alarm* feed is implemented with a similar pattern.
 
-The connection-based architecture is the highest performance option -- but it's far more difficult to operate and maintain than the polling and hook architectures.   The provider service must be production-quality: it must be highly-available and fault-tolerant.    
+The connection-based architecture is the highest performance option -- 
+but imposes more overhead on operations compared to the polling and hook architectures.   
```

Comment:
Changes LGTM, you tell me if you want more 👀  to review it or call ready for press.